apps/callscreen/js/calls_handler.js

```diff
@@ -209,7 +204,8 @@ var CallsHandler = (function callsHandler() {
   function removeCall(index) {
     handledCalls.splice(index, 1);
 
-    if (handledCalls.length === 0) {
+    if ((handledCalls.length === 0) ||
+        (handledCalls[0].call.state === 'disconnected')) {
```

Comment:
I've noticed that my change doesn't fix the problem for other scenarios too involving conference calls. I think I'll just remove this change and put it in a follow-up.